Projects/App/Sources/Screens/ProfileScreen/Screens/DeleteAccount/Slack/SlackBot.swift

```diff
@@ -31,10 +31,11 @@ class SlackBot {
                 "Authorization": "Bearer xoxb-" + result.slackDetails.token,
                 "Content-Type": "application/json",
             ]
-
+            
+            let stagingSlackChannelId: String = "C03L32WF6JD"
             let requestBody = self.generatePostMessageBody(
```

Comment:
Shouldn't we do this in giraffe instead, feels a bit dangerous that we are sending the messages, because then anyone could just grab the token and send whatever to that channel.

Reply:
the token comes from giraffe. This id is for the slack channel. The prod channel's id is on giraffe. I'll add the staging one as well